src/theme/components/accordion.ts

```diff
@@ -1,21 +1,28 @@
 const containerStyles = {
   borderLeftWidth: "1px",
   borderRightWidth: "1px",
-  borderColor: "ui.gray.medium",
   width: "100%",
+  _hover: {
+    borderLeftWidth: "1px",
+    borderRightWidth: "1px",
+    borderColor: "ui.gray.dark",
+    // The bottom border of the accordion item being hovered
+    // over is hidden beneath the top border of the following
+    // sibling, so we need to grab and change that border, as well.
+    "& + .chakra-accordion__item": {
+      borderTopColor: "ui.gray.dark",
+    },
+  },
 };
 const buttonStyles = {
   border: "none",
   fontWeight: 500,
-  _hover: {
```

Comment:
I think you still need the `_hover` rule but, according to ticket 862, the background should be removed. Chakra seems to have its own default hover bg color. Try making it "transparent" so it removes it rather than "ui.white". I have a feeling that this won't work for the warning and error bg colors though.... So this might be tricky and I think ticket 862 should be updated to reflect the newly added bg colors.

@bigfishdesign13 what do you think?

Reply:
DSD-862 has been updated to clarify the color assignments.

Reply:
I don't. Since this is not taking the `__css` prop, separate style props are fine. Make `colorMap[content.accordionType]`  into a variable though so it only does the lookup once.

Reply:
Will do.